api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -300,7 +300,7 @@ public Order getOrderByOrderNumber(String orderNumber) {
 	 * @see org.openmrs.api.db.OrderDAO#getNextOrderNumberSeedSequenceValue()
 	 */
 	@Override
-	public Long getNextOrderNumberSeedSequenceValue() {
+	public synchronized Long getNextOrderNumberSeedSequenceValue() {
```

Comment:
@haripriya999  what did you intend to do ??

Reply:
i get. but since we get a  Travis failure , it seems thats not the solution. Can you get another solution??

Reply:
@mozzy11 , I am trying to figure out on what can be done :)

Reply:
would you like to suggest something from your side? :)